data/customers.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Okay I guess.... I would make the `customer_id` an explicit identifier to the customers entity